### ISSUE TITLE
 rabbitmq-server dependencies

### ISSUE THREAD

**Issue by poteirard** [kleiram/vagrant-symfony#29]
Hi !

I just add the submodule as it says in the documentation. I did vagrant up in the "vagrant" folder and I just have this message:

````
TASK: [rabbitmq | Update apt again] *******************************************
ok: [default]

TASK: [rabbitmq | Install RabbitMQ] *******************************************
failed: [default] => {"failed": true}
stderr: E: Unable to correct problems, you have held broken packages.

stdout: Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 rabbitmq-server : Depends: erlang-nox (>= 1:16.b.3) but it is not going to be installed or
                            esl-erlang but it is not installable

msg: '/usr/bin/apt-get -y -o "Dpkg::Options::=--force-confdef" -o "Dpkg::Options::=--force-confold"   install 'rabbitmq-server'' failed: E: Unable to correct problems, you have held broken packages.


FATAL: all hosts have already failed -- aborting

PLAY RECAP ********************************************************************
           to retry, use: --limit @/Users/poteirard/site.retry

default                    : ok=34   changed=27   unreachable=0    failed=1

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
`````
Using:
Vagrant 1.8.1
VirtualBox: 5.0.10

Any ideas how to solve this? 

Thanks

**Comment by kleiram** [comment]
Hi there!

That's a new one! I'll look into it. Maybe the setup for RabbitMQ changed...

**Comment by mablae** [comment]
@kleiram Have a look at https://github.com/kleiram/vagrant-symfony/pull/30

**Comment by poteirard** [comment]
Thanks guys :+1:

**Comment by kleiram** [comment]
Fix with PR #30!
